internal/text/testing.go

```diff
@@ -34,9 +35,8 @@ func OpRemoveSummaryLineElapsedTime(line string) string {
 	return line
 }
 
+var trailingElapsedTimePattern = regexp.MustCompile(`\s*\(\d+(\.\d+)?m?s\)\s*$`)
+
 func OpRemoveTestElapsedTime(line string) string {
-	if i := strings.Index(line, " (0."); i > 0 && i+8 == len(line) {
-		return line[:i]
-	}
-	return line
+	return trailingElapsedTimePattern.ReplaceAllString(line, "")
```

Comment:
I don't think I see why this change was made. I generally try to avoid regex. Reverting this still has the tests passing. 

I'm going to revert this part of the change for now. We can always re-introduce it later if it fixed something that I'm missing.

Reply:
The time was being reported in `ms` rather than seconds so it didn't start with 0.  Sounds like changing `GOTESTSUM_FORMAT` will fix that.